js/libs/mime_mapper.js

```diff
@@ -0,0 +1,135 @@
+/* exported MimeMapper */
```

Comment:
simple lib we use in SMS as well; this makes it easy to "guess" which mime type we have given a file extension.